backend/src/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Sleep API</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
```

Comment:
I don't think we should have a description meta for this, nor should we have open graph headers. We don't want to risk sharing this on social media and baiting uses with a link that's broken.